sriovnet.go

```diff
@@ -413,3 +414,24 @@ func GetVfIndexByPciAddress(vfPciAddress string) (int, error) {
 	}
 	return -1, fmt.Errorf("vf index for %s not found", vfPciAddress)
 }
+
+func GetNetDeviceFromPci(vfPciAddress string) (string, error) {
+	var devName string
+	vfDir := filepath.Join(PciSysDir, vfPciAddress, "net")
+	_, err := os.Lstat(vfDir)
+	if err != nil {
+		return devName, fmt.Errorf("cannot get a network device with pci address %v %q", vfPciAddress, err)
+	}
+
+	dirContents, err := ioutil.ReadDir(vfDir)
+	if err != nil || len(dirContents) < 1 {
+		return devName, fmt.Errorf("failed to get network device name in %v %v", vfDir, err)
+	}
+
+	if len(dirContents) < 1 {
+		return devName, fmt.Errorf("no network device found in %v", vfDir)
+	}
+
+	devName = dirContents[0].Name() // assuming one net device in this directory
```

Comment:
@moshe010, 
pci device supports multiple names and API should return array to support so.

few examples of such multiple netdev are:
1. multiple ipoib devices created using different pkey.
2. dev_port based multiple netdevices per pci device.